bid/views.py

```diff
@@ -35,6 +31,23 @@
 from django.dispatch import receiver
 from django.utils.timezone import now
 
+from django.http import JsonResponse
+from .models import Property
+from .models import Winner
+from .models import Auction
+from .models import WinnerRating
+from .models import User
+from .models import ShippingGift
+from django.http import HttpResponse
+from django.views.decorators.csrf import csrf_exempt
+import json
+from django.middleware.csrf import get_token
+from django.conf import settings
+import os
+from django.views.decorators.http import require_http_methods
+from django.views.decorators.http import require_GET
+from rest_framework_simplejwt.tokens import AccessToken
```

Comment:
run `pip freeze > requirements.txt` to save the required packages
<img width="575" alt="Screenshot 2024-04-09 at 4 30 00 PM" src="https://github.com/yunchipang/goodhome/assets/48413155/e301c5ed-cfa8-40a2-8bec-33356127745d">